src/hssm/likelihoods/analytical.py

```diff
@@ -322,45 +258,35 @@ def logp_ddm_sdv(
     rt = pt.abs(data[:, 0])
     response = data[:, 1]
     flip = response > 0
-    a = a * 2
+    a = a * 2.0
     v_flipped = pt.switch(flip, -v, v)  # transform v if x is upper-bound response
     z_flipped = pt.switch(flip, 1 - z, z)  # transform z if x is upper-bound response
     rt = rt - t
-    p = ftt01w(rt, a, z_flipped, err, k_terms)
-
-    # This step does 3 things at the same time:
-    # 1. Computes f(t|v, a, z) from the pdf when setting a = 0 and z = 1.
-    # 2. Computes the log of above value
-    # 3. Computes the integration given the sd of v
-    logp = (
-        pt.log(p + epsilon)
-        + (
-            (a * z_flipped * sv) ** 2
-            - 2 * a * v_flipped * z_flipped
-            - (v_flipped**2) * rt
-        )
-        / (2 * (sv**2) * rt + 2)
-        - pt.log(sv**2 * rt + 1 + epsilon) / 2
-        - 2 * pt.log(a + epsilon)
-    )
-    logp = pt.where(rt <= 0, OUT_OF_BOUNDS_VAL, logp)
-    checked_logp = check_parameters(
-        logp,
-        sv >= 0,
-        msg="sv >= 0",
+
+    p = pt.maximum(ftt01w(rt, a, z_flipped, err, k_terms), pt.exp(OUT_OF_BOUNDS_VAL))
+
+    logp = pt.where(
+        rt <= epsilon,
+        OUT_OF_BOUNDS_VAL,
+        pt.log(p)
+        - v_flipped * a * z_flipped
+        - (v_flipped**2 * rt / 2.0)
+        - 2.0 * pt.log(a),
     )
-    checked_logp = check_parameters(checked_logp, a >= 0, msg="a >= 0")
-    # checked_logp = check_parameters(checked_logp, 0 < z < 1, msg="0 < z < 1")
-    # checked_logp = check_parameters(checked_logp, np.all(rt > 0), msg="t <= min(rt)")
+
+    checked_logp = check_parameters(logp, a >= 0, msg="a >= 0")
+    checked_logp = check_parameters(checked_logp, z >= 0, msg="z >= 0")
+    checked_logp = check_parameters(checked_logp, z <= 1, msg="z <= 1")
     return checked_logp
 
 
-def logp_ddm(
+def logp_ddm_sdv(
     data: np.ndarray,
     v: float,
     a: float,
     z: float,
     t: float,
+    sv: float,
     err: float = 1e-7,
```

Comment:
default k_terms to 20 here
if this makes it very slow we can rethink, but it could definitely help make it more rebust.